lib/latte.ts

```diff
@@ -0,0 +1,482 @@
+// Latte supports Mocha syntax for now, but we'd like to evolve.
```

Comment:
@Hehk i'm curious if this conversion helped us learn more about zen... 
Did this conversion shed any light on why zen needs a separate version of latte, and can't just use `mocha` directly?

Reply:
After reviewing, nothing in here stands out to me as "obviously we can't use mocha because ____"